config/tsconfig.base.json

```diff
@@ -18,16 +20,21 @@
 		"moduleResolution": "Bundler",
 
 		// Emit
+		"declaration": true,
 		"declarationMap": true,
+		"sourceMap": true,
 
 		// Interop Constraints
 		"esModuleInterop": true,
 		"forceConsistentCasingInFileNames": true,
+		"isolatedModules": true,
+		"verbatimModuleSyntax": false, // @todo Enable this.
 
 		// Language and Environment
 		"jsx": "react",
 		"lib": [ "ES2022", "DOM" ],
-		"target": "ES2018",
+		"moduleDetection": "force",
+		"target": "ES2018", // @todo bump to ES2022
```

Comment:
Is there a reason _not_ to bump this before 0.1.0?

Reply:
Nope. Bump it.